### ISSUE TITLE
[Accessibility] Localization of "This is the single point tooltip template" string in activity-timeline.component.html

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4982]
# Summary

## Describe the bug
The string "This is the single point tooltip template" on line 23 in the `activity-timeline.component.html` file needs to be localized to support multiple languages.

## To Reproduce
1. Navigate to the `apps/web-mzima-client/src/app/activity/activity-timeline/activity-timeline.component.html` file.
2. Locate the following code block on line 23:
```html
<ng-template #tooltipTemplate let-model="model">
  This is the single point tooltip template
  <pre>{{ model | json }}</pre>
</ng-template>
```

## Expected behavior
The "This is the single point tooltip template" string should be localized and retrieved from a translation file or service, so that it can be displayed in the user's preferred language.

## How to solve
1. In the `activity-timeline.component.html` file, replace the hardcoded "This is the single point tooltip template" string with a translation key (something like., `{{ 'ACTIVITY.TOOLTIP_TEMPLATE' | translate }}`).

## Screenshots
<img width="1200" alt="Screenshot 2024-07-26 at 14 57 11" src="https://github.com/user-attachments/assets/07d84158-f474-4c60-9bef-6ea401f51b33">
